tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -631,9 +633,10 @@ class RemapperFuseConvWithBias : public RemapperTest {
  public:
   template <int dim, DataType DTYPE>
   void RunTest() {
-      if (DTYPE == DT_BFLOAT16 &&
-        !(IsMKLEnabled() && mkl_op_registry::IsBF16SupportedByOneDNNOnThisCPU()) ) {
-        GTEST_SKIP() << "*** skipping, bf16 not supported ***";
+    if (DTYPE == DT_BFLOAT16 &&
+        !(IsMKLEnabled() &&
+          mkl_op_registry::IsBF16SupportedByOneDNNOnThisCPU())) {
+      GTEST_SKIP() << "onednn not enabled or bf16 not supported on this CPU";
```

Comment:
Nit: Maybe replace `onednn` with `oneDNN` (all places in this file)?
```suggestion
      GTEST_SKIP() << "oneDNN not enabled or bf16 not supported on this CPU";
```